renderers/glfw/windowing.lisp

```diff
@@ -268,8 +268,11 @@
 (defmethod (setf window:min-size) :before (min-size (window window))
   (set-window-size-limits window min-size (window:max-size window)))
 
+(defmethod get-window-bool-attribute ((window window) attribute)
```

Comment:
I'd much prefer this to be just a plain function. There's no real reason for it to be a generic function.

Reply:
Was My initial attempt - then went into character saving mode.

Since it's more of a glfw wrapper than an actual window method, should I then have it accept the window object or rather a pointer?

Reply:
yea your function is fine, just change it to `defun get-window-bool-attribute (window attribute)`